src/Analytics.js

```diff
@@ -234,6 +239,11 @@ class Analytics {
         this._setVisitVariable('RTE: Uses Richtext Mode', state ? 'on' : 'off');
     }
 
+    setBreadcrumbs(state) {
+        if (this.disabled) return;
+        this._setVisitVariable('Breadcrumbs', state ? 'enabled' : 'disabled');
```

Comment:
Is this safe to call repeatedly from componentDidUpdate? It pushes to an array...

Reply:
I believe so. It didn't blow up in testing, and matomo is built for much busier applications than us.

Reply:
Let me rephrase that: is this going to send telemetry every time LeftPanel's `componentDidUpdate` method is called? If so, this doesn't seem like the right event, I'd assume we want to send telemetry upon a breadcrumbs related user interaction?

Reply:
Oh... Yup, we want to know when it changes not every time something happens. Will fix.

Reply:
turns out that the arguments to didUpdate are also the old state, not the new state: https://reactjs.org/docs/react-component.html#componentdidupdate